doc/api/tracing.md

```diff
@@ -5,11 +5,24 @@
 Trace Event provides a mechanism to centralize tracing information generated by
 V8, Node core, and userspace code.
 
-Tracing can be enabled by passing the `--trace-events-enabled` flag when starting a
-Node.js application.
+Tracing can be enabled by passing the `--trace-events-enabled` flag when
+starting a Node.js application.
 
 The set of categories for which traces are recorded can be specified using the
-`--trace-event-categories` flag followed by a list of comma separated category names.
+`--trace-event-categories` flag followed by a list of comma separated category
```

Comment:
The flag should be named `--trace-events-categories`  for consistency.

Reply:
This flag already exists and is not introduced by this PR